ui/projekt-monsterz-front/.gitignore

```diff
@@ -0,0 +1,44 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
A separate .gitignore for the UI is unnecessary, since we have one at the root of the project.